### PR TITLE
[#166220931] Changes unknown ente beneficiario from "?" to empty string

### PR DESCRIPTION
**Pull request by Undermaken**
This PR aims to remove "?" on payment confirmation when "ente benificiario" is unknown

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#166220931](https://www.pivotaltracker.com/story/show/166220931): Come CIT sulla conferma di pagamento se manca l'ente beneficiario non voglio vedere nulla

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ class ConfirmPaymentMethodScreen extends React.Component<Props, never> {
           <PaymentBannerComponent
             currentAmount={currentAmount}
             paymentReason={paymentReason}
-            recipient={recipient || UNKNOWN_RECIPIENT}
+            recipient={recipient || UNKNOWN_RECIPIENT_EMPTY}
```
Comment (fpersico) [inline ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx]: Can't we just make "recipient" prop optional and check it in PaymentBannerComponent?

Comment (fpersico) [inline ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx]: Can't we just make "recipient" prop optional and check it in PaymentBannerComponent?

Comment (Undermaken) [reply ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx]: @fpersico PaymentBannerComponent is used in different part of the app (3 usage)
I make this change only for ConfirmPaymentMethodScreen as requested.
I don't know if in the other parts we want the same behaviour

If you think so I can change the code as suggested

Comment (fpersico) [reply ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx]: I think the behaviour must be the same in all the screen. What you think @matteodesanti?

**Comment by Undermaken** [comment]
check (eventually uniform) if all screens act with same behaviour

File: ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx (hunk 1 above)
Comment (fpersico) [reply ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx]: @Undermaken

> Can't we just make "recipient" prop optional and check it in PaymentBannerComponent?

**Pull request merged**